### ISSUE TITLE
How to train the model

### ISSUE THREAD

**Issue by NIEYALI** [longcw/faster_rcnn_pytorch#1]
thanks for your share, But how to train the model, would you share the files?

Thank you very much

**Comment by longcw** [comment]
The training script has been uploaded.  You can follow the README.md to prepare data and the pre-trained model. You may need to tune the loss function defined in `faster_rcnn/faster_rcnn.py` by yourself.

**Comment by abhiML** [comment]
What do you mean tune the loss function?
